rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+from typing import Any, Mapping, Type
+
+import ray
+
+from ray.rllib.core.rl_module.rl_module import RLModule, ModuleID
+from ray.rllib.core.optim.rl_optimizer import RLOptimizer
+from ray.rllib.policy.sample_batch import MultiAgentBatch
+from ray.air.config import ScalingConfig
+from ray.train._internal.backend_executor import BackendExecutor
+from ray.train.torch import TorchConfig
```

Comment:
we should import these only import one of these backends depending framework: str

Reply:
done